release-team/role-handbooks/ci-signal/upgrade-job-versions.md

```diff
@@ -0,0 +1,137 @@
+# Upgrade Job Versions
+
+An example of how, given an upgrade job, the different ways I can verify:
+- what version is it upgrading FROM (old-version)
+- what version is it upgrading TO (new-version)
+
+## TL;DR
+- almost no jobs actually use a released/tagged build 
+- most jobs use one of these versions, built off the HEAD of a branch
+
+| name           | branch eg    | actual eg                           |
+| -------------- | ------------ | ----------------------------------- |
+| ci/latest      | master       | v1.13.0-alpha.0.1226+426ef9d349bb3a |
+| ci/k8s-beta    | release-1.12 | v1.12.0-beta.1.129+1d58f1aebfe1e3   |
+| ci/k8s-stable1 | release-1.11 | v1.11.3-beta.0.71+a4529464e4629c    |
+| ci/k8s-stable2 | release-1.10 | v1.10.8-beta.0.33+a963fce72fed31    |
+| ci/k8s-stable3 | release-1.9  | v1.9.11-beta.0.36+06bf888123ca89    |
+
+NB: these examples are based on the current cycle at time of writing, which is release-1.12
+
+### A: Look at the job name
```

Comment:
Thanks a lot for elaborating so clearly Aaron. 1 ques - are A B and C different ways if getting at the same information? or does results from previous steps feed into the next (doesnt seem so)? Worth clarifying may be?

Reply:
>An example of how, given an upgrade job, the different ways I can verify

Should the headings not have sequential letters? I was trying to order from simplest to hardest

Reply:
>An example of how, given an upgrade job, the different ways I can verify

Should the headings not have sequential letters? I was trying to order from simplest to hardest

Reply:
Ah I missed the text "different ways I can verify". May be call them "Option A" etc?